docs/manual/docs/cmd/spo/apppage/apppage-update.md

```diff
@@ -0,0 +1,29 @@
+# spo apppage update
+
+Updates the single-part app page
+
+## Usage
+
+```sh
+spo apppage update [options]
```

Comment:
The command description says `spo apppage set` I would suggest we use set to mach a pattern we follow with the CLI where we use add,set,remove,get for most of the commands. What do you think?

Issue desc here: [https://github.com/pnp/office365-cli/issues/875](https://github.com/pnp/office365-cli/issues/875)